<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Печать этикеток</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="renderHints">
        <set>QPainter::TextAntialiasing</set>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::RubberBandDrag</enum>
       </property>
       <property name="resizeAnchor">
        <enum>QGraphicsView::NoAnchor</enum>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::FullViewportUpdate</enum>
       </property>
       <property name="rubberBandSelectionMode">
        <enum>Qt::IntersectsItemShape</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="pageDefault">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Элемент не выбран</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_preview"/>
    <addaction name="actionExport_PDF"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Правка</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Добавить</string>
     </property>
     <addaction name="actionRectangle"/>
     <addaction name="actionBarcode"/>
     <addaction name="actionText"/>
     <addaction name="actionImage"/>
     <addaction name="actionEllipse"/>
     <addaction name="actionLine"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionZoom_fit"/>
   </widget>
   <widget class="QMenu" name="menuPrint_option">
    <property name="title">
     <string>Опции печати</string>
    </property>
    <addaction name="actionPrint_outlines"/>
    <addaction name="actionPrint_crop_marks"/>
    <addaction name="actionAnti_aliasing"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuPrint_option"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionLine"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionText"/>
   <addaction name="actionImage"/>
   <addaction name="actionBarcode"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionZoom_fit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/filenew.png</normaloff>:/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="toolTip">
    <string>Новая этикетка</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/filesave.png</normaloff>:/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/button_cancel.png</normaloff>:/images/button_cancel.png</iconset>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/glabels-box.png</normaloff>:/images/glabels-box.png</iconset>
   </property>
   <property name="text">
    <string>Прямоугольник</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionBarcode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/glabels-barcode.png</normaloff>:/images/glabels-barcode.png</iconset>
   </property>
   <property name="text">
    <string>Штрих код</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionText">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/glabels-text.png</normaloff>:/images/glabels-text.png</iconset>
   </property>
   <property name="text">
    <string>Текст</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/button_cancel.png</normaloff>:/images/button_cancel.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/fileprint.png</normaloff>:/images/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPrint_preview">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/fileprint.png</normaloff>:/images/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Предварительный просмотр</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/glabels-image.png</normaloff>:/images/glabels-image.png</iconset>
   </property>
   <property name="text">
    <string>Изображение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/glabels-ellipse.png</normaloff>:/images/glabels-ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Эллипс</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/zoomin.png</normaloff>:/images/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/zoomout.png</normaloff>:/images/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_fit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/viewmagfit.png</normaloff>:/images/viewmagfit.png</iconset>
   </property>
   <property name="text">
    <string>Подогнать размер</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Enter</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/glabels-line.png</normaloff>:/images/glabels-line.png</iconset>
   </property>
   <property name="text">
    <string>Линия</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionExport_PDF">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/exportpdf.png</normaloff>:/images/exportpdf.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт PDF</string>
   </property>
  </action>
  <action name="actionAnti_aliasing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сглаживание</string>
   </property>
  </action>
  <action name="actionPrint_outlines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Печать контуров</string>
   </property>
  </action>
  <action name="actionPrint_crop_marks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Печать разметки</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
